models/index.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
better have this in separate function. Something like 
```
const getSequelizeInstance = (config) => {
  if (config.use_env_variable) {
    return new Sequelize(process.env[config.use_env_variable], config);
  } else {
    return new Sequelize(
      config.database,
      config.username,
      config.password,
      config
    );
  }
}
```
This would allow to get rid of `let`. Using `let` is bad manners by default, especially using it on top level.